nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EvaluateXPath.java

```diff
@@ -199,8 +216,11 @@ protected void init(final ProcessorInitializationContext context) {
     }
 
     @OnScheduled
-    public void initializeXPathFactory() throws XPathFactoryConfigurationException {
+    public void initializeFactory() throws XPathFactoryConfigurationException {
         factoryRef.set(XPathFactory.newInstance(NamespaceConstant.OBJECT_MODEL_SAXON));
+        final DocumentBuilderFactory docBuilderFactory=DocumentBuilderFactory.newInstance();
```

Comment:
With the new changes, these lines were removed but I'll watch out for spaces around '='